fml/backtrace_unittests.cc

```diff
@@ -19,18 +19,21 @@ TEST(BacktraceTest, CanGatherBacktrace) {
     auto trace = BacktraceHere(0);
     ASSERT_GT(trace.size(), 0u);
     ASSERT_NE(trace.find("Frame 0"), std::string::npos);
```

Comment:
Should these verify that `CanGatherBacktrace` is in the string?

Reply:
On master, the frame 0 of offset 0 is actually `CanGatherBacktrace`. I fixed the PR and added the test.

Reply:
Some CI tests seems to run without symbols. I added skipping the test `BacktraceStartsWithCallerName` if the function name in the first line of stacktrace is "Unknown".

Reply:
I tried to make all CIs pass, but only on 
Linux Unopt, execinfo's `backtrace` returns the backtrace which contains itself, and it makes hard to write this test.
I'll remove this test from PR.